t/ui/27-plugin_obs_rsync.t

```diff
@@ -45,21 +45,55 @@ $t->get_ok('/login');
 
 BAIL_OUT('Login exit code (' . $t->tx->res->code . ')') if $t->tx->res->code != 302;
 
-$t->get_ok('/admin/obs_rsync')->status_is(200, "index status")->element_exists('a[href*="/admin/obs_rsync/Proj1"]')
-  ->content_unlike(qr/script\<\/a\>/);
+sub _el {
+    my ($project, $run, $file) = @_;
+    return qq{a[href="/admin/obs_rsync/$project/runs/$run/download/$file"]} if $file;
+    return qq{a[href="/admin/obs_rsync/$project/runs/$run"]}                if $run;
+    return qq{a[href="/admin/obs_rsync/$project"]};
+}
 
-$t->get_ok('/admin/obs_rsync/Proj1')->status_is(200, "project status")->element_exists('[rsync_iso.cmd]')
-  ->element_exists('[rsync_repo.cmd]')->element_exists('[openqa.cmd]');
+sub _el1 {
+    my ($project, $file) = @_;
+    return _el($project, '.run_last', $file);
+}
```

Comment:
The names `_el` and `_el1` are not very descriptive at all, which will make future work harder.